crates/sui-graphql-rpc/src/types/base64.rs

```diff
@@ -10,6 +10,7 @@ use fastcrypto::encoding::Encoding as FastCryptoEncoding;
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub(crate) struct Base64(pub(crate) Vec<u8>);
 
+/// A Base64 encoded string.
```

Comment:
i feel like this comment is not needed

Reply:
Similar comment here as above that it's better than "No description", but also nit, it's not a "Base64 encoded String", it's a "String containing Base64-encoded binary data".